src/app/pages/PerpetualPage/hooks/usePerpetual_ContractDetails.ts

```diff
@@ -49,7 +50,13 @@ export const usePerpetual_ContractDetails = (pairType: PerpetualPairType) => {
     };
 
     get24hVolume().catch(console.error);
-  }, [pair.id]);
+    const intervalId = setInterval(
+      () => get24hVolume().catch(console.error),
+      5 * 60 * 1000,
```

Comment:
Maybe you can create a global constant for 1 minute and use it here and in `GraphQLProvider` as well (you wouldn't need comments then).

Reply:
We could move this to a constant as well, but I don't think a constant for one minute in ms is really needed.